spec/restaurant_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'customer'
+
+describe 'restaurant' do
+
+  subject {Restaurant.new}
+
+  it 'creates a restaurant' do
+    expect(subject).to be_kind_of Restaurant
+  end
+
+    it 'creates a menu' do
+    expect(subject.menu).to be_kind_of Array
+  end
+
+    it 'menu contains dishes when created' do
+    expect(subject.menu[0]).to be_kind_of Hash
+  end
+
+   it 'dishes has a name' do
+    expect(subject.menu[0][:name]).to eq "Fried rice"
+  end
+
+     it 'dishes has a price' do
+    expect(subject.menu[0][:price]).to eq 65
+  end
```

Comment:
Lint/BlockAlignment: `end` at 25, 2 is not aligned with `it 'dishes has a price' do` at 23, 5